vault/core.go

```diff
@@ -165,6 +165,18 @@ type raftInformation struct {
 	joinInProgress      bool
 }
 
+type migrationInformation struct {
+	// seal to use during a migration operation. It is the
+	// seal we're migrating *from*.
+	seal        Seal
+	masterKey   []byte
+	recoveryKey []byte
+	// newRecoveryKey is used is the master key when shamir seal is in use. This
```

Comment:
Extra "is".

Reply:
Also, it's not the master key, you're still thinking of pre-1.3 shamir seals.  It's the combined unseal key, the master key is on disk.

Reply:
You are right. Renamed the field as `shamirCombinedKey`. If you have a better suggestion, I'm open for it.